src/components/emailJS-example/EmailJSExampleOfUse.jsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
nie ptrzebujemy komponentu tylko przykład użycia w czystym JS. Zamiast formularzy w html i inputów podaj konkretne zmienne w JS

Reply:
Nie za bardzo rozumiem, co masz na myśli. 
Poprawki z pozostałych komentarzy naniesione, ale to nie wiem jak zrobić.